packages/desktop-client/src/components/transactions/TransactionsTable.jsx

```diff
@@ -562,17 +564,40 @@ function PayeeCell({
         }
       }}
       formatter={() => getPayeePretty(transaction, payee, transferAcct)}
-      unexposedContent={props => (
-        <>
-          <PayeeIcons
-            transaction={transaction}
-            transferAccount={transferAcct}
-            onNavigateToTransferAccount={onNavigateToTransferAccount}
-            onNavigateToSchedule={onNavigateToSchedule}
-          />
-          <UnexposedCellContent {...props} />
-        </>
-      )}
+      unexposedContent={props => {
+        const payeeInfo = (
+          <div style={{ display: 'flex', alignItems: 'center', width: '100%' }}>
+            <PayeeIcons
+              transaction={transaction}
+              transferAccount={transferAcct}
+              onNavigateToTransferAccount={onNavigateToTransferAccount}
+              onNavigateToSchedule={onNavigateToSchedule}
+            />
+            <UnexposedCellContent
+              {...props}
+              style={{ borderBottom: `1px dashed ${theme.pageTextSubdued}` }}
```

Comment:
Shouldn't this only be underlined if there is an importedPayee?

Reply:
https://github.com/actualbudget/actual/pull/3018#issuecomment-2222878509

Reply:
Sorry, as I was eating I only just realised what you meant. Will fix.